landingcorp/css/_template.scss

```diff
@@ -3,3 +3,5 @@ $primary-color: red;
 body {
   background-color: $primary-color;
 }
+
+
```

Comment:
Please run `npm run fmt` (or `npx prettier --write .`) from the project root directory in order to format all relevant source.